plugins/ltac/tactic_option.ml

```diff
@@ -9,39 +9,90 @@
 (************************************************************************)
 
 open Libobject
-open Pp
+
+type tac_option_locality =
+  | Default
+  | Global
+  | Local
+  | Export
+  | GlobalAndExport
+
+let tac_option_locality =
+  let open Attributes in
+  let open Attributes.Notations in
+  let one x = attribute_of_list [x,single_key_parser ~name:"Locality" ~key:x ()] in
+  one "local" ++ one "global" ++ one "export" >>= fun ((local,global),export) ->
+  match local, global, export with
+  | None, None, None -> return Default
+
+  | Some (), Some (), _ -> CErrors.user_err Pp.(str "Cannot combine local and global.")
+  | Some (), _, Some () -> CErrors.user_err Pp.(str "Cannot combine local and export.")
+
+  | Some (), None, None -> return Local
+  | None, Some (), None -> return Global
+  | None, None, Some () -> return Export
+  | None, Some (), Some () -> return GlobalAndExport
+
+let warn_default_locality =
+  CWarnings.create ~name:"deprecated-tacopt-without-locality" ~category:"deprecated" Pp.(fun () ->
+      strbrk
+        "The default and global localities for this command outside \
+         sections are currently equivalent to the combination of \
+         \"global\" (with the meaning used by the \"Set\" command), \
```

Comment:
Is this really necessary to explain the standard meaning of `global` by comparing it to another command? What about just say "the standard meaning of `global` as defined in the reference manual"? (https://coq.inria.fr/refman/language/core/modules.html#coq:attr.global)